cocos/editor-support/cocosbuilder/CCBAnimationManager.cpp

```diff
@@ -99,7 +99,11 @@ Node* CCBAnimationManager::getRootNode()
 
 void CCBAnimationManager::setRootNode(Node *pRootNode)
 {
+    CC_SAFE_RELEASE_NULL(_rootNode);
+    
     _rootNode = pRootNode;
+    
+    CC_SAFE_RETAIN(_rootNode);
```

Comment:
@andyque OK,but I can not add it now. I need some time to download the project and I need to wait until after work to do